L1Trigger/L1THGCal/test/HGCalTriggerBestChoiceTester.cc

```diff
@@ -26,14 +26,24 @@
 #include "DataFormats/ForwardDetId/interface/ForwardSubdetector.h"
 #include "DataFormats/ForwardDetId/interface/HGCTriggerDetId.h"
 
+#include "SimDataFormats/CaloHit/interface/PCaloHit.h"
+#include "SimDataFormats/CaloHit/interface/PCaloHitContainer.h"
+#include "SimDataFormats/CaloTest/interface/HGCalTestNumbering.h"
+#include "SimDataFormats/CaloTest/src/HGCalTestNumbering.cc"
+#include "Geometry/HGCalGeometry/interface/HGCalGeometry.h"
+#include "Geometry/CaloTopology/interface/HGCalTopology.h"
+
 #include "L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h"
 #include "L1Trigger/L1THGCal/interface/HGCalTriggerFECodecBase.h"
 #include "L1Trigger/L1THGCal/interface/fe_codecs/HGCalBestChoiceCodecImpl.h"
 
 #include <stdlib.h> 
+#include <map> 
 #include "TH2.h"
 
 using namespace std;
+using namespace edm;
```

Comment:
Avoid using namespaces globally. Use edm:: and reco:: instead. Ideally also for std.